base/abstractarray.jl

```diff
@@ -806,6 +806,34 @@ pointer{T}(x::AbstractArray{T}, i::Integer) = (@_inline_meta; unsafe_convert(Ptr
 # If the subtype hasn't defined the required method, it falls back to the
 # _getindex function again where an error is thrown to prevent stack overflows.
 
+"""
+    getindex(A, inds...)
+
+Returns a subset of array `A` as specified by `inds`, where each `ind` may be an
```

Comment:
I think the preferred style is to describe the action in the imperative. In this case it would be "return" instead of "returns."